src/sessions/index.js

```diff
@@ -0,0 +1,198 @@
+import { loadSessions, createSession, removeSession } from './session';
+import { UnknownSessionError, UnknownIdentityError } from '../utils/errors';
+
+class Sessions {
+    #sessions;
+    #storage;
+    #identities;
+
+    #identityListeners = {};
```

Comment:
Can we make this a Map for performance reasons?

Reply:
For future reference: https://mathiasbynens.be/notes/shapes-ics